src/main/java/seedu/masslinkers/logic/parser/ParserUtil.java

```diff
@@ -27,7 +29,12 @@
  */
 public class ParserUtil {
 
-    public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_INVALID_INDEX = "Index is missing or not a non-zero unsigned integer.";
+    public static final String MESSAGE_UNEXPECTED_CHARACTERS = "There appears to be extraneous characters "
```

Comment:
We could define `n/`, `g/`, `t/` as parameter prefixes in the UG under add command? 
Then, the error could phrase it as 
```Input with missing parameter prefixes found```
cuz extraneous characters may not be that clear?

Reply:
Currently:
edit 1 yoyoyooyo gives ```
There appears to be extraneous characters being supplied to the command.
Refer to help for the command format.
```
while 
add dsdxisjiji gives
`Invalid arguments 'dsdxisjiji' provided to the command!`
even though both seem to be of similar nature error.

Reply:
Fixed ✅